docs/integrations/dydx.mdx

```diff
@@ -9,42 +9,41 @@ title: dYdX
 
 **A:** Here is a quick (5-10 minutes) step-by-step breakdown. Please reach out to us at [skip.money/discord](http://skip.money/discord) if you have questions!
 
-#### **Download the Slinky binary.**
+#### **Download the Connect binary.**
 
-The best way to get the Slinky binary is in the GitHub releases page for Slinky.
+The best way to get the Connect binary is in the GitHub releases page for Connect.
 
 https://github.com/skip-mev/slinky/releases
 
-The **minimum** slinky version required for dYdX is [`v1.0.1`](https://github.com/skip-mev/slinky/releases/tag/v1.0.1)
+The **minimum** connect version required for dYdX is [`v1.0.1`](https://github.com/skip-mev/slinky/releases/tag/v1.0.1)
 
 We also provide a container image at [ghcr.io/skip-mev/slinky-sidecar](https://github.com/skip-mev/slinky/pkgs/container/slinky-sidecar/230279145?tag=v1.0.1)
 
-This will include the Slinky binary, `slinky`.
+This will include the Connect binary, `slinky`.
 
-#### **Integrate the Slinky sidecar into your setup.**
+#### **Integrate Connect into your setup.**
 
 The configuration of your validator setup requires you to tackle a few problems which we’ll mention here.
 
-**_Configure the Slinky process_**
+**_Configure the Connect process_**
 
-To run the slinky sidecar with stable defaults defined by the skip team
+To run connect with stable defaults defined by the skip team
 
 ```bash
 slinky --marketmap-provider dydx_api --market-map-endpoint https://<dydx-node REST API endpoint>
 ```
 
-> Notice, some default values may need to change depending on how you've setup your node + slinky-sidecar. See the **marketmap-provider** [section](/validator/flags-guide#marketmap-provider) of the [flags-guide](/validator/flags-guide)
+> Notice, some default values may need to change depending on how you've setup your node + connect instance. See the **marketmap-provider** [section](/validator/flags-guide#marketmap-provider) of the [flags-guide](/validator/flags-guide)
```

Comment:
is this okay to change? seems like slinky-sidecar is a code reference instead of a mention

Reply:
wym?

Reply:
![CleanShot 2024-07-05 at 15 54 15](https://github.com/skip-mev/slinky/assets/29874473/72f00970-2235-47e6-83ea-9df359f97515)

just this, unsure if it's a super big deal though